jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -146,21 +143,21 @@ YUI.add('user-profile', function() {
       @param {Object} data The data from the request.
     */
     _fetchEntitiesCallback: function(type, error, data) {
+      // Turn off the loader, regardless of success or error.
+      if (type === 'charm') {
```

Comment:
Why did you split this out of the conditional below? I don't really see any advantage to it being up here with the exception of if there is an error it'll still turn off the loading indicators. If so could you update the code comment plz